server/apps/todo/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+from django.utils.timezone import now
+from model_utils import Choices
+from ..common.model_utils import TimestampedModel, CreatorInfoModel
+
+from ..users.models import User
+
+
+class Todo(TimestampedModel, CreatorInfoModel):
+    COMPLETED = "completed"
+    INPROGRESS = "inprogress"
+    STATUS = Choices((COMPLETED, COMPLETED), (INPROGRESS, INPROGRESS))
+    status = models.CharField(choices=STATUS, default=INPROGRESS, max_length=20)
+    owner = models.CharField(max_length=100)
```

Comment:
Owner would be a User

Reply:
Updated. Changed owner as a ForeignKey to the User model